examples/subplots_axes_and_figures/custom_figure_class.py

```diff
@@ -1,27 +1,55 @@
 """
 ===================
-Custom Figure Class
+Custom Figure class
```

Comment:
subclass?

Reply:
Maybe just "Subclassing Figure"?

Reply:
fine with me too.

Reply:
Any preference? The second one is shorter the first one is more precise.

Reply:
"Custom Figure subclasses"?